wallet-webapps-admin/src/main/webapp/vue-app/components/admin/wallets/WalletAdminWalletsTab.vue

```diff
@@ -406,8 +401,6 @@ export default {
       const walletTableHeaders = this.walletHeaders.slice();
       if (!this.contractDetails) {
         walletTableHeaders.splice(4, 1);
```

Comment:
walletTableHeaders.splice() repeated 2 times

Reply:
Yes, that was to remove two elements from the array when the contract is not loaded.